docs/general/standalone-support.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Standalone Support
+description: Describes how games can receive standalone support, as well as the conditions- and requirements.
+---
+
+# Standalone Games
+
+We offer support for Standalone games, which do not use an emulator to connect to RetroAchievements. This is done by directly making HTTP calls to our emulator integration API (the "Connect API"), be it through directly modifying the game or creating a mod that has the code required to connect to the RetroAchievements server. However, all Standalone support must abide strict requirements, as well as approval by the RetroAchievements Admins. Even if all the guidelines below are met, this will not guarantee we will accept your proposal. For the best chances of success, it is vital that you present a highly-detailed plan to the admins.
+
+[[toc]]
+
+## What Are the Requirements to Work on a Standalone
+
+Standalone support cannot be proposed by just about anyone, as there are strict non-negotiable requirements that must be met.
+
+- The proposer must either be a Developer on RetroAchievements, or a Junior Developer with at least one published set. As these achievement sets still have to follow our global [Achievement Design](https://docs.retroachievements.org/developer-docs/achievement-design.html) rules. This is not something we can trust anyone with who has not showed us their proficiency in set design before.
+   - Alternatively, the proposer may suggest a standalone without having developed a set, but must do so with the support of a RetroAchievements Developer who will be in charge of set design.
+- You must be able to enforce Hardcore restrictions as shown in the [Global Leaderboard and Achievement Hunting Rules](https://docs.retroachievements.org/guidelines/users/global-leaderboard-and-achievement-hunting-rules.html). Such examples include black/whitelisting other mods that reduce the challenge of achievements, or disallowing multiplayer to prevent unfair advantages. These advantages should only be reserved for Softcore players, if applicable.
```

Comment:
It may also be good to mention disabling/blocking in-game cheats in this area, as it's probable the standalone developer has direct access to this capability.